src/modules/CreateDrops/DropSummary/DropSummary.tsx

```diff
@@ -11,6 +11,7 @@ import { DropSummaryModal } from '../DropSummaryModal';
 import { SummaryItemImage, SummaryItemText } from './SummaryItem';
 
 interface DropSummaryProps {
+  mode: 'token' | 'nft';
```

Comment:
I think the parent component should pass in the Button text 🤔

Reply:
i dont think that looks clean 🤔 , but if you prefer that one i'll make the change!

Reply:
hmm i think there will be more than 3 types of drops in future

Reply:
updated!